integration-tests/utils/integration-utils.js

```diff
@@ -391,10 +391,24 @@ const logPageResponses = page => {
     'googleapis',
     'bvdp'
   ]
-  const handle = res => {
-    const request = res.request()
-    if (terraRequests.some(urlPart => request.url().includes(urlPart))) {
-      console.log('page.http', `${request.method()} ${res.status()} ${res.url()}`)
+  const handle = response => {
+    const request = response.request()
+    const url = request.url()
+    const shouldLogRequest = terraRequests.some(urlPart => url.includes(urlPart))
+    if (shouldLogRequest) {
+      const method = request.method()
+      const status = response.status()
+      console.log('page.http', `${method} ${status} ${url}`)
+
+      const isErrorResponse = status >= 400
+      if (isErrorResponse) {
+        const responseIsJSON = response.headers()['content-type'] === 'application/json'
+        response.text().then(content => {
+          console.log('page.http.error', `${method} ${status} ${url}`, responseIsJSON ? JSON.parse(content) : content)
+        }).catch(err => {
+          console.error('page.http.error', 'Unable to get response content', err)
```

Comment:
Q: Do we also want to have `console.error(response.text())` in catch block because an error thrown by L#405 - 407 of code would be different from the response.

Reply:
The catch block here is catching an error throw by `reponse.text`.